web/views.py

```
@@ -6485,3 +6487,163 @@ def delete_post(request, post_id):
+    context_object_name = "rooms"
+
+    def get_queryset(self):
+        return VoiceChatRoom.objects.filter(is_active=True)
+
+
+class VoiceChatCreateView(LoginRequiredMixin, CreateView):
+    """View for creating a new voice chat room."""
+
+    model = VoiceChatRoom
+    template_name = "web/voice_chat/create.html"
+    fields = ["name"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)